server/task.go

```diff
@@ -53,6 +53,14 @@ func (s *Server) registerTaskRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to update task").SetInternal(err)
 		}
 
+		issueFind := &api.IssueFind{
```

Comment:
nit: since this is only used for creating activity for UpdateSchema task, I would move it close to that block